<launch>
	<node name="dmp" pkg="dmp" type="dmp_server" respawn="false" output="screen"/>
</launch>
